README.md

```diff
@@ -8,26 +8,13 @@ My personal portfolio site
 
 - HTML
 - CSS
+- JAVASCRIPT
 
 
 ## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
 ## Authors
 
 👤 **Cindy Dorantes**
```

Comment:
[ OPTIONAL ]
- [ ] Great you provided the link to the `MIT.md` file but it seems to be missing. Either add it to your project to remove the link entirely because leaving broken links in your `README.md` does make your repo professional. I kindly suggest you fix that